src/viewPages/ArticleDetailed.jsx

```diff
@@ -45,8 +37,32 @@ export default function ArticleDetailed() {
           <p>
             Created at: {new Date(selectedArticle.created_at).toLocaleString()}
           </p>
-          <p>Votes: {selectedArticle.votes} </p>
-          <p>Comments: {selectedArticle.comment_count}</p>
+          <p>
+            <button
+              onClick={() => {
+                setIsLoading(true);
+                const voteChange = isLiked ? -1 : 1;
+                patchApi(`/api/articles/${id_article}`, { inc_votes: voteChange })
+                  .then((response) => {
+                    setSelectedArticle(response.article);
+                    setIsLoading(false);
+                    setIsLiked(!isLiked);
+                  })
+                  .catch(() => {
+                    setIsLoading(false);
+                    setError(true);
+                  });
+              }}
+            >
+              {isLiked ? 'Unlike' : 'Like'}
```

Comment:
Consider moving the api call to it's own reusable function rather than having it all within the `onClick`.

Also the page looks like it refreshes/resets when I press the like/unlike button. This isn't an ordinary behaviour which would happen when a user clicks a button such as 'like'. Consider optimistic rendering instead, notes available here: [https://notes.northcoders.com/courses/js-front-end/ui-behaviours-and-optimistic-rendering](url)